exercises/luhn/description.md

```diff
@@ -7,8 +7,11 @@ The task is to write a function that checks if a given string is valid.
 
 Validating a Number
 ------
+Strings of length 1 or less are not valid. Spaces are allowed in the input,
+but they should be stripped before checking. All other non-digit characters
```

Comment:
this sentence makes me think of other cases: how about a string of only spaces?

However, it's not true that I would refuse to see this merged unless such a case was added. What do people think of it?